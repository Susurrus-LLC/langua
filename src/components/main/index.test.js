import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'

import Main from '.'

describe('Main', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<Main>Test</Main>, div)
    ReactDOM.unmountComponentAtNode(div)
  })

  it('renders accurately', () => {
    let tree = renderer.create(<Main>Test</Main>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
