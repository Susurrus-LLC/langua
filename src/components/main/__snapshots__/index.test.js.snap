// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main renders accurately 1`] = `
<main
  className="main"
>
  Test
  <div
    className="Toastify"
  />
</main>
`;
